examples/llm/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*!
+# LLM Example Application
+
+This example application runs a large language model in an application's service.
+
+# How It Works
+
+Models and tokenizers are served locally using a local Python server. They are expected
+at `model.bin` and `tokenizer.json`.
+
+The application's service exposes a single GraphQL field called `prompt` which takes a prompt
+as input and returns a response.
+
+When a prompt is submitted, the application's service uses the `get_blob_from_url`
```

Comment:
Minor: I think the API name changed

```suggestion
When a prompt is submitted, the application's service uses the `fetch_url`
```